src/input_cp2k_dft.F

```diff
@@ -225,9 +225,12 @@ SUBROUTINE create_dft_section(section)
                           "towards a spin-polarized solution. "// &
                           "Thus, larger tolerance increases chances of ending up "// &
                           "with spin-polarization. "// &
-                          "More details in a GitHub `thread <https://github.com/cp2k/cp2k/issues/4389>`. "// &
                           "This option is only valid for unrestricted (i.e. spin polarised) "// &
-                          "Kohn-Sham (UKS) calculations.", &
+                          "Kohn-Sham (UKS) calculations. It also needs non-zero "// &
+                          "FORCE_EVAL/DFT/.../ADDED_MOS to actually affect the calculations, "// &
```

Comment:
For the input descriptions you can use most of the [Markdown syntax](https://github.com/cp2k/cp2k/blob/master/docs/README.md#syntax-cheat-sheet) as for the rest of the manual:

```suggestion
                          "[ADDED_MOS](#CP2K_INPUT.FORCE_EVAL.DFT.SCF.ADDED_MOS) to actually affect the calculations, "// &
```